TypeCobol.Test/Parser/Programs/TypeCobol/ProcedureCall-ParamCompatibily.rdzMix.txt

```diff
@@ -28,75 +30,93 @@
-Line 65[12,41] <27, Error, Syntax> - Syntax error : Function 'proc2' expected parameter 'param1' of usage  and received 'picNum10Comp5' of usage NativeBinary
+Line 73[12,41] <27, Error, Syntax> - Syntax error : Function 'proc2' expected parameter 'param1' with picture X(10) and received 'picNum10Comp5' with picture 9(10)
+Line 73[12,41] <27, Error, Syntax> - Syntax error : Function 'proc2' expected parameter 'param1' of usage  and received 'picNum10Comp5' of usage NativeBinary
            call proc2 input picNum10Comp5
       *    KO
-Line 67[12,35] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' to be NOT an array and received 'tabNum5' which is an array
+Line 75[12,35] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' to be NOT an array and received 'tabNum5' which is an array
            call proc1 input tabNum5
       *    KO
-Line 69[12,37] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' with picture 9(10) and received 'tabAlpha5' with picture X(10)
-Line 69[12,37] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' to be NOT an array and received 'tabAlpha5' which is an array
+Line 77[12,37] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' with picture 9(10) and received 'tabAlpha5' with picture X(10)
+Line 77[12,37] <27, Error, Syntax> - Syntax error : Function 'proc1' expected parameter 'param1' to be NOT an array and received 'tabAlpha5' which is an array
            call proc1 input tabAlpha5
       *OK because we only target one element of the array
       *TODO not working yet
       *    call proc1 input tabAlpha5(1)
 
 
       *KO
-Line 76[12,36] <27, Error, Syntax> - Syntax error : Function 'proc3' expected parameter 'param1' synchonized and received 'picNum10' not synchronized
-           call proc3 input picNum10
+Line 84[12,37] <27, Error, Syntax> - Syntax error : Function 'proc3' expected parameter 'param1' synchronized and received 'picNum10' not synchronized
+           call proc3  input picNum10
+Line 85[12,37] <27, Error, Syntax> - Syntax error : Function 'proc31' expected parameter 'param1' synchronized (Left) and received 'picNum10' not synchronized
+           call proc31 input picNum10
+Line 86[12,37] <27, Error, Syntax> - Syntax error : Function 'proc32' expected parameter 'param1' synchronized (Right) and received 'picNum10' not synchronized
+           call proc32 input picNum10
       *OK
-           call proc3 input picNum10Sync 
+           call proc3  input picNum10Sync
+           call proc31 input picNum10SyncLeft
+           call proc32 input picNum10SyncRight
+           call proc3  input picNum10Sync
+           call proc31 input picNum10SyncLeft
+           call proc32 input picNum10SyncRight
+           call proc3  input picNum10Sync
+           call proc31 input picNum10SyncLeft
+           call proc32 input picNum10SyncRight
```

Comment:
I may be missing something, but it seems like the same test is repeated 3 times :
```cob
      *   Ok
           call proc3  input picNum10Sync
           call proc31 input picNum10SyncLeft
           call proc32 input picNum10SyncRight
```
As for error tests we should also have
```cob
      *   Ko
           call proc3  input picNum10SyncLeft
           call proc31 input picNum10SyncRight
           call proc32 input picNum10Sync
           call proc3  input picNum10SyncRight
           call proc31 input picNum10Sync
           call proc32 input picNum10SyncLeft
```

Reply:
No ! Look at the definitions, `proc3`, `proc31` and `proc32` are different, they declare different parameters with either SYNC, SYNC LEFT or SYNC RIGHT so these are 3 different tests.

Reply:
I think the same test (from line 88->90) is repeated again twice, that is from line 91->93 and again 94->96.

Reply:
@rooksdo You're right ! Sorry I mixed up the combinations :man_facepalming:. Fixed with 2e6c1c04a671c2fcb3e38ab3bee786792a2bc939.